eslint.config.mjs

```diff
@@ -0,0 +1,5 @@
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default hmppsConfig({
+  extraIgnorePaths: ['assets'],
```

Comment:
not sure about this but has started failing eslint checks.  Will investigate in next PR

Reply:
I wasn't sure about this for the nomis-sync-dashboard - please tag me in your next PR if you're investigating